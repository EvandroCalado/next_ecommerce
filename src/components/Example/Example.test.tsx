import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import { Example } from '.';

describe('Example', () => {
  it('should render default', () => {
    const { container } = render(<Example />);

    expect(
      screen.getByRole('heading', { name: 'Example' }),
    ).toBeInTheDocument();

    expect(container).toMatchSnapshot();
  });
});
