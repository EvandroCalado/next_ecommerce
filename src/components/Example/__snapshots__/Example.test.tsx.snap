// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example should render default 1`] = `
<div>
  <div>
    <h1>
      Example
    </h1>
  </div>
</div>
`;
